Neos.RedirectHandler/Configuration/Settings.yaml

```diff
@@ -0,0 +1,17 @@
+Neos:
+ RedirectHandler:
+  features:
+    hitCounter: true
+  statusCode:
+    'redirect': 301
+    'gone': 410
+
+TYPO3:
+  Flow:
+    http:
+      chain:
+        'process':
+          chain:
+            'redirection':
+              position: 'before routing'
```

Comment:
Adding the redirection *before* the routing means that it will be triggered almost for every request.
In some (most?) scenarios it would be enough to trigger the redirect only if the existing routes didn't match a request - for those cases the redirection http component could be triggered only if the routing component didn't succeed:
```
$routingMatchResults = $componentContext->getParameter(RoutingComponent::class, 'matchResults');
if ($routingMatchResults !== NULL) {
	return;
}
// ..
```